build.gradle.kts

```
@@ -215,6 +231,8 @@ tasks {
 		useJUnitPlatform()
 		filter {
 			includeTestsMatching("*Tests")
+			// included for documentation purposes
+			// includeTestsMatching("*Demo")
```

Is this correct?

```suggestion
			includeTestsMatching("*Demo")
```

Other than that, I think we can merge.

---

Nope. This is just dead code.